modules/compliance-tests/src/smithy4s/compliancetests/internals/package.scala

```diff
@@ -29,4 +33,33 @@ package object internals {
       h.++(Hints(smithy.api.TimestampFormat.EPOCH_SECONDS.widen))
     )
   }
+
+  def splitQuery(queryString: String): (String, String) = {
+    queryString.split("=", 2) match {
```

Comment:
if the queryString is empty, this must yield `Array()` and you'd get a pattern match error, is that possible?

Reply:
Ah I guess this does not happen because we're mapping on a list of string